base/applications/cmdutils/fc/lang/pl-PL.rc

```diff
@@ -1,41 +1,45 @@
+/*
+Translation by Piotr Hetnarowicz <piotrhwz@gmail.com> (April, 2021)
+*/
```

Comment:
I'd say adding these headers is not necessary, because you can see all contributors with Git history and Git blame.